assessments/NWEA_Map/earthmover.yaml

```diff
@@ -8,12 +8,14 @@ config:
   memory_limit: 1GB
   state_file: ./runs.csv
   show_graph: False
+  tmp_dir: ${TEMPORARY_DIRECTORY}
   parameter_defaults:
     STUDENT_ID_NAME: Student_StateID
     STUDENT_ID_FROM: Student_StateID
     STUDENT_ID_XWALK: ''
     STUDENT_ID_QUERY: ''
     DATABASE_CONNECTION: ''
+    INPUT_FILETYPE: csv
```

Comment:
Add filetype customization to allow non-standard (i.e., Parquet) files to be used as input.